demo/docker-compose.yml

```diff
@@ -101,5 +101,20 @@ services:
     networks:
       - tmail
 
+  apisix:
+    command: >
+      bash -c "cat /apisix-config/config.yaml > /usr/local/apisix/conf/config.yaml
+      && ln -s /apisix-config/apisix.yaml /usr/local/apisix/conf/apisix.yaml 
+      && /docker-entrypoint.sh docker-start"
```

Comment:
Question: We can not mount directly these two files? (APISIX override them on startup?)

Reply:
I just re-tested, the mount volume work.
I will update commit 

// 
TLDR: When I set `APISIX_STAND_ALONE=true`, I got some errors related to the mounted file, permission, override...
So I manually set the value file in this way.
But now, we don't need env `APISIX_STAND_ALONE` :D